zerver/worker/queue_processors.py

```diff
@@ -423,11 +423,11 @@ class TestWorker(QueueProcessingWorker):
     # and appends it to a file in /tmp.
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-        fn = settings.ZULIP_WORKER_TEST_FILE
-        message = ujson.dumps(event)
-        logging.info("TestWorker should append this message to %s: %s" % (fn, message))
-        with open(fn, 'a') as f:
-            f.write(message + '\n')
+        fn = settings.ZULIP_WORKER_TEST_FILE  # nocoverage
+        message = ujson.dumps(event)  # nocoverage
+        logging.info("TestWorker should append this message to %s: %s" % (fn, message))  # nocoverage
+        with open(fn, 'a') as f:  # nocoverage
+            f.write(message + '\n')  # nocoverage
```

Comment:
You can put `# nocoverage` comments on a block, e.g. the `def consume` block, rather than on every line.  Adding it to every line is messier than just writing a test for the thing :)

Reply:
I merged a version of this change that just touches the `def` line; you can drop this commit.